example-local/autorun/autorun.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env bash
```

Comment:
this is a more portable approach
also, consider using spellcheck when doing scripting